docs/reference_guides/apps/grid_integration/Bidder.rst

```diff
@@ -13,6 +13,155 @@ uncertain price scenario has a corresponding power output. As shown in the figur
 each of these uncertain price and power output pairs formulates a segment in the
 bidding curves.
 
+Here we present a stochastic bidding model for a renewable integated energy system (Wind generator + PEM).
+
+
+Day-Ahead Bidding Problem for Wind + PEM IES
+---------------------------------------------
+
+The objective function is the expected profit, which equals the revenue substracts the cost. 
+We want to consider the revenue from the electricity market and the hydrogen market.
+
+.. math:: \max \quad \sum_{s \in S, t \in T^{DA}}\omega_{s}[{(\pi_{t,s}^{DA}P_{t,s}^{DA} + \pi_{t,s}^{RT}(P_{t,s}^{RT}-P_{t,s}^{DA}))\Delta t  + Pr^{H}m_{t,s}^{H}- c_{t,s}}] - C_{fix}
+
+s.t.
+
+.. math:: P_{t,s}^{DA} \leq P_{t,s}^{RT} \quad \forall t, s \quad \quad (1)
+.. math:: P_{t,s} = P^{RT}_{t,s} \quad \forall t, s \quad \quad (2)
+.. math:: (\pi_{t,s'}^{DA} - \pi_{t,s}^{DA})(P_{t,s'}^{DA} - P_{t,s}^{DA}) \geq 0 \quad \forall s \in S, \forall s' \in S \backslash s, \forall t \in T^{DA} \quad \quad (3)
+.. math:: P_{t,s}^{DA} \leq P_{t,s}^{wind} \quad \forall t, s \quad \quad (4)
+.. math:: P_{t,s}^{RT} \leq P_{t,s}^{wind} \quad \forall t, s \quad \quad (5)
+.. math:: P_{t,s}^{wind} \leq f_{t} P_{max}^{wind} \quad \forall t, s \quad \quad (6)
+.. math:: P_{t,s}^{DA} + P_{t,s}^{PEM} \leq P_{max}^{wind} \quad \forall t, s \quad \quad (7)
```

Comment:
This constraint makes sense but I'm wondering if this constraint (combined also with 17) is flexible enough to allow for the use case where the PEM is used in RT when the plant's DA bid is accepted but in RT the grid doesn't actually take the option of that DA power. Or perhaps other cases... @bknueven any thoughts?

Reply:
Yes, eqn 7 has an typo. I will fix that. 

I think we may increase the weight of the hydrogen revenue at the DA bidding so that the electricity bid will be more conservative. In such case, we can reduce the underbidding. But the cost is we will lose some revenue from DA market.